zoo.py

```diff
@@ -0,0 +1,127 @@
+class Animal:
+    """ Creates Animal object with a name and a list of preys. """
```

Comment:
Another way might be:
'An Animal has a name, and likes eating certain other species of animal.'